test/index.js

```diff
@@ -1002,7 +1048,7 @@ describe('request()', function () {
             setTimeout(function () {
 
                 res.writeHead(200);
-                res.write(payload);
+                res.write(internals.payload);
```

Comment:
I've changed this line because tests were failing with message:
```
Multiple callbacks or thrown errors received in test "request() handles read timeout" (error): payload is not defined
      at null._onTimeout (C:\workspace\wreck\test\index.js:1051:27)
      at Timer.listOnTimeout [as ontimeout] (timers.js:110:15)
```